pipeline_steps/kibana.groovy

```diff
@@ -15,7 +15,7 @@ def kibana(branch, vm=null){
 
 def kibana_prep(branch){
   dir("kibana-selenium") {
-    git url: "https://github.com/rcbops-qe/kibana-selenium.git", branch: "${branch}"
+    git url: env.KIBANA_SELENIUM_REPO, branch: "${branch}"
```

Comment:
We should create a job for that, then anything else that needs to happen at the same time can be added to it.